src/toM3u8.js

```diff
@@ -206,13 +206,29 @@ const organizeCaptionServices = (captionServices) => captionServices.reduce((svc
     return svcObj;
   }
 
-  svc.forEach(({channel, language}) => {
+  svc.forEach((service) => {
+    const {
+      channel,
+      language
+    } = service;
+
     svcObj[language] = {
       autoselect: false,
       default: false,
       instreamId: channel,
       language
     };
+
+    if (service.hasOwnProperty('aspectRatio')) {
```

Comment:
Is there any reason not to add all properties?

Reply:
If we add them when not present, the output of both 608 and 708 will include things like `aspectRatio`, where we want them only for 708.

Reply:
Well I meant add any properties that `service` has.

Reply:
the three extra props that it has are copied over. Channel and language are already accounted for above.